addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4610,7 +4610,7 @@ export class OdooEditor extends EventTarget {
                     }
                     this.powerbox.open(commands);
                 } else {
-                    this._applyCommand('insert', this._createLink(text, url));
+                    this._applyCommand('insert', targetSupportsHtmlContent ? this._createLink(text, url) : text);
```

Comment:
- Is not there many stuff in the main `else` which do not check `targetSupportsHtmlContent` but should? The code just above seems to be adding a while video iframe ^^
- Commit message: "then the link content and the link URL can be adapted in each translation" => I don't think you can until we implement task-3430333, right?

Reply:
O_O... About the commit message, I actually ended up in a situation where I got this:

![image](https://github.com/odoo/odoo/assets/71644421/55594a6b-6ec2-46bd-88cf-f646b2bdb666)
But I do not manage to reproduce it anymore...